doc/classes/Resource.xml

```diff
@@ -19,6 +19,21 @@
 				Override this method to return a custom [RID] when [method get_rid] is called.
 			</description>
 		</method>
+		<method name="_setup_local_to_scene" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Override this method to customise the newly duplicated resource created from [method PackedScene.instantiate], if the original's [member resource_local_to_scene] is set to [code]true[/code].
+				[b]Example:[/b] Set a random [code]damage[/code] value to every local resource from an instantiated scene, excluding the original.
```

Comment:
What "excluding the original" refers to? I did a test with this method in a tool script and every Resource instance has this called.

Reply:
"Excluding the original" refers to the resource every local resource has been duplicated from. The method is called on every new one, but the original Resource, basically the following in your scene, is not affected by this:

```Ini
[sub_resource type="Resource" id="Resource_gw2eg"]
resource_local_to_scene = true
script = ExtResource("1_khad7")
damage = 0
```

You may actually change this Resource at run time without affecting the duplicates. However, every **newly** instantiated scene will create its own local resources with the original properties in mind. 
This is worth knowing especially for Resources like Stylebox, or resources saved to disk.

Under the hood you may If you also feel like this is necessary information and/or have better ways to phrase it, feel free to suggest.

Reply:
But if I open HolderOfLocal.tscn and save it, it will be modified. If I run the scene and print the value, it will also have a different value.
```GDScript
@export var res: Resource

func _ready() -> void:
	print(res.damage)
```
So when this method doesn't get called?

Reply:
That is... unexpected. Oh dear.
For now I'll remove the "except" stuff, but that feels like it should not happen. Maybe this bug, if it is one, only happens when a Resource is contained within a PackedScene?